src/libData/BlockData/BlockHeader/TxBlockHeader.cpp

```diff
@@ -169,6 +174,11 @@ const TxnHash & TxBlockHeader::GetTxRootHash() const
     return m_txRootHash;
 }
 
+const StateHash & TxBlockHeader::GetStateRootHash() const
```

Comment:
:bulb: Consider defining this method where it is declared (improves efficiency with no apparent downside).

Reply:
Sure.

Reply:
For such changes though could you rather make generalized github issues. May be as we grow someone from the community can pick them up. We could tag such enhancements something so that they can be found.